packages/atomic/src/components/common/generated-answer/rephrase-buttons.tsx

```diff
@@ -73,7 +73,7 @@ export const RephraseButtons: FunctionalComponent<RephraseButtonProps> = (
               <div class="icon-container text-neutral-dark h-full mx-auto shrink-0 relative">
                 <atomic-icon icon={option.icon}></atomic-icon>
               </div>
-              <div class="rephrase-btn-label hidden text-neutral-dark">
+              <div part="rephrase-btn-label" class="hidden text-neutral-dark">
```

Comment:
```suggestion
              <div part="rephrase-button-label" class="hidden text-neutral-dark">
```

We don't normally use abbreviation for parts name.
Parts are kinda like public options/props on components, and so their name must be complete and explicit.